fluent/test/context_test.js

```diff
@@ -7,19 +7,16 @@ import FluentBundle from '../src/bundle';
 import FluentResource from '../src/resource';
 
 suite('Bundle', function() {
-  let bundle, args, errs;
+  let bundle;
 
-  setup(function() {
-    errs = [];
-  });
-
-  suite('addMessages', function(){
+  suite('addResource', function(){
     suiteSetup(function() {
       bundle = new FluentBundle('en-US', { useIsolating: false });
-      bundle.addMessages(ftl`
+      let resource = new FluentResource(ftl`
```

Comment:
Do we need this resource?

Reply:
Are you asking whether this should be a separate variable?

Reply:
Yeah.

Reply:
I see. Yeah, I can inline this one as well. Thanks.